eslint/babel-eslint-config-internal/index.js

```diff
@@ -15,7 +13,7 @@ module.exports = {
   },
   env: {
     node: true,
-    es2020: true,
+    es2022: true,
```

Comment:
Per https://eslint.org/docs/latest/user-guide/configuring/language-options#specifying-environments, `es2022: true` implies `parserOptions.ecmaVersion: 2022`, so I removed the former option.